classwork/сlasswork2.1/main.cpp

```diff
@@ -0,0 +1,108 @@
+#include <iostream>
+#include <unordered_map>
+#include <fstream>
+#include <string>
+#include <print>
+#include <format>
+
+namespace {
+    constexpr const char* DEFAULT_FILE_NAME = "user_stats.txt";
+    constexpr const char* RESET_COMMAND = "bread";
+    constexpr const char* DELETE_COMMAND = "delete";
+}
+
+class UserStatistics {
+public:
+    explicit UserStatistics(const std::string& statisticsFilePath = DEFAULT_FILE_NAME)
+            : fileName(statisticsFilePath) {
+        loadFromFile();
+    }
+
+    ~UserStatistics() {
+        saveToFile();
+    }
+
+    bool incrementCounterForUser(const std::string& name, std::string& message) {
+        if (name == RESET_COMMAND) {
```

Comment:
it is better to provide separate function to reset database. at the moment your function violates single responsibility principle - it does multiple actually unrelated things and as a result your implementation doesn't match the function name. Let's increment inside increment function not reset everything